pkg/private/deb/make_deb.py

```diff
@@ -30,6 +31,8 @@
 
 from pkg.private import helpers
 
+Multiline = Enum('Multiline', ['NO', 'YES', 'YES_ADD_NEWLINE'])
```

Comment:
Why are you making a tri-state when there are only 2. They are either 1 line, or they are multiline and all in the same file. Having the YES state just forces us to have tests for something that doesn't get used by the code.

Reply:
Description in the control file is the third state (see line 166).
We can treat both multiline scenarios as the same thing but that was exactly what my previous attempt was. In that case we end up with strings that start with spaces and logic that needs to trim extra spaces in certain scenarios.

Please state which approach do you prefer and what changes would you like to see to get that landed.

Reply:
The core problem is that you're trying to improve on a quick hack which was badly thought through 10+ years ago.  There are a few distinct cases  which were shoehorned into one method.  One is the "description + list of elements".  Another is single line   Name: value.    The final was the "First line is on the field line and the rest is reflowed multiline."    Since the concepts were never clear, people (including myself) have been implementing this wrong since the beginning.

Making this more maintainable is low priority for me right now, so let's just take yours as is. If it ends up breaking users who have relied on the previous behavior, I'll figure something out at that time.